docs/presentation/Content/ContentInstancesOfSpecificClasses.md

```diff
@@ -2,38 +2,161 @@
 
 > TypeScript type: [ContentInstancesOfSpecificClassesSpecification]($presentation-common).
 
-Returns content for instances of specific ECClasses.
+Specification which allows targeting content instances based on their ECClass.
```

Comment:
for -> from - "A specification that creates content from instances of specific ECClasses."?

Reply:
"from" suggests we're creating content from a single instance. However, we're creating it for a single instance and it may be related to other instances that we're going to include as well. So I'd say we're creating content for a single instance from possibly multiple instances :)